xgi/drawing/draw.py

```diff
@@ -1707,111 +1771,154 @@ def draw_dihypergraph(
         edge_marker_size = _interp_draw_arg(
             edge_marker_size, settings["min_node_size"], settings["max_node_size"]
         )
-    if rescale_sizes and isinstance(lines_lw, np.ndarray):
-        lines_lw = _interp_draw_arg(
-            lines_lw, settings["min_lines_lw"], settings["max_lines_lw"]
+    if rescale_sizes and isinstance(dyad_lw, np.ndarray):
+        dyad_lw = _interp_draw_arg(
+            dyad_lw, settings["min_dyad_lw"], settings["max_dyad_lw"]
         )
 
     # parse colors
-    lines_fc, lines_c_mapped = _parse_color_arg(lines_fc, H_conv.edges)
+    dyad_color, dyads_c_mapped = _parse_color_arg(dyad_color, H.edges)
 
     # convert numbers to colors for FancyArrowPatch
-    if lines_c_mapped:
+    if dyads_c_mapped:
         norm = mpl.colors.Normalize()
         m = cm.ScalarMappable(norm=norm, cmap=edge_marker_fc_cmap)
-        lines_fc = m.to_rgba(lines_fc)
-
-    # compute the augmented projection
-    # this is a networkx Graph as a bipartite representation of H_conv
-    # nodes and hyperedges > 2 are represented by nodes
-    G_aug = _augmented_projection(H_conv)
+        dyad_color = m.to_rgba(dyad_color)
 
-    phantom_nodes = [n for n in list(G_aug.nodes) if n not in list(H_conv.nodes)]
-
-    if pos is None:  # compute positions, first based on original nodes
-        pos_nodes = barycenter_spring_layout(H_conv)
-        # then compute the barycenter positions for the "edge" phantom nodes
-
-        centers = {}
-        for idx, members in DH.edges.members(dtype=dict).items():
-            center = np.zeros((2,))
-            edge_bi_id = phantom_nodes[idx]
-            for node in members:
-                center += pos_nodes[node]
-            centers[edge_bi_id] = center / len(members)
-
-        # merge both pos dictionaries
-        pos_nodes.update(centers)
-        pos = pos_nodes
-
-        # recompute positions from the original positions
-        pos = spring_layout(G_aug, pos=pos, iterations=iterations)
-
-    # draw "node" nodes
     ax, node_collection = draw_nodes(
-        H=H_conv,
-        pos=pos,
+        H=H,
+        pos=node_pos,
         ax=ax,
         node_fc=node_fc,
         node_ec=node_ec,
         node_lw=node_lw,
         node_size=node_size,
-        node_fc_cmap=node_fc_cmap,
         node_shape=node_shape,
-        zorder=4,
-        params=settings,
-        node_labels=node_labels,
+        node_fc_cmap=node_fc_cmap,
+        zorder=2,
+        params=node_settings,
+        node_labels=None,
         rescale_sizes=rescale_sizes,
+        **kwargs,
     )
 
-    # draw "edge" phantom nodes
-    if edge_marker_toggle:
+    if node_labels is not None:
+        draw_node_labels(
+            H,
+            pos=node_pos,
+            node_labels=node_labels,
+            ax_nodes=ax,
+            zorder=3,
+        )
```

Comment:
this can be passed simply to draw_nodes like before, right?

Reply:
Yep! Good catch!